examples/set-namespace-matcher-mode/README.md

```diff
@@ -0,0 +1,54 @@
+# set-namespace: Matcher Mode Example
+
+### Overview
+
+This example demonstrates how to run [`set-namespace`] function in Matcher Mode.
```

Comment:
Please add link to the related section in the usage doc.

Reply:
done.